src/main/java/group1/mips_simulator/components/Value.java

```diff
@@ -17,13 +17,18 @@ public Value(int value) {
 
     //region getters and setters
     public short get() {
-        return value_;
+        return this.getShort();
     }
 
+    public short getShort() {return value_;}
+    //public int getInt() { return (int) value_;} //unimportant
```

Comment:
You don't like my `getInt()` 😿 
That's ok hahah. Instead of leaving it in as a comment go ahead and just remove it.